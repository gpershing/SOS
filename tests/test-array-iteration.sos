x: int = 1
y: int = 3
z: int = 8

many_x: array int = [x, 4, y, 6, 3, z]

printing: (x: int) -> void = print(x)

printing(many_x)
