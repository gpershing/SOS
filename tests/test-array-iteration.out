1
4
3
6
3
8